### PR TITLE
Fix README example

### PR DESCRIPTION
**Pull request by RHavar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # bitcoinjs-message
 
-[![TRAVIS](https://secure.travis-ci.org/bitcoinjs/bitcoin-message.png)](http://travis-ci.org/bitcoinjs/bitcoin-message)
```
Comment (dcousens) [inline README.md]: Why remove TRAVIS?
It seems the url is wrong, should be `/bitcoinjs-message` instead.

File: README.md [hunk 2]
```diff
@@ -10,7 +9,7 @@
 
 ``` javascript
 var bitcoin = require('bitcoinjs-lib')
-var message = require('bitcoinjs-message')
+var bitcoinMessage = require('bitcoinjs-message')
```
Comment (dcousens) [inline README.md]: Sure

**Comment by dcousens** [comment]
Just a heads up,  for now, you can still use `bitcoin.message` (aka, `bitcoinjs-lib`) for this functionality.
Its only intended to be removed by `3.0.0`, see https://github.com/bitcoinjs/bitcoinjs-lib/pull/456

**Comment by fanatid** [comment]
this is not actual after #2

**Pull request closed without merging**